presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -385,6 +386,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Experimental: enable pushdown of range filters on subscripts (a[2] = 5) into ORC column readers",
                         hiveClientConfig.isRangeFiltersOnSubscriptsEnabled(),
                         false),
+                booleanProperty(
+                        ADAPTIVE_FILTER_REORDERING_ENABLED,
+                        "Experimental: enable adaptive filter reordering",
+                        true,
```

Comment:
Do we wanna have a hiveClientConfig to flag the default value?

Reply:
@highker Sure. Added `hive.adaptive-filter-reordering-enabled` configuration property.